pkg/overrides/csv_placeholders_test.go

```
@@ -0,0 +1,232 @@
+	return out
+}
+
+func helmImageOverrideKeys(t *testing.T, templatesDir string) []string {
+	t.Helper()
+	re := regexp.MustCompile(`imageOverrides\.([A-Za-z0-9_]+)`)
+	seen := map[string]struct{}{}
+	err := filepath.Walk(templatesDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() || !(strings.HasSuffix(path, ".yaml") || strings.HasSuffix(path, ".yml")) {
```

golangci-lint (2.13.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)